LayoutTests/fast/forms/datetimelocal/datetimelocal-editable-components/up-down-arrow-starts-at-current-year-expected.txt

```diff
@@ -0,0 +1,11 @@
+Tests that pressing the up or down arrow on the year field of a date input starts the value at the current year.
+
+On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
+
+
+PASS new Date(input1.value).getFullYear() is 2026
+PASS new Date(input2.value).getFullYear() is 2024
```

Comment:
We need to write the test in such a way that it doesn't output the actual year, otherwise we'll need to rebaseline once a year 😅